src/actions/createProject.js

```diff
@@ -13,4 +13,6 @@ export default async function createProject(projectName) {
   );
 
   await createDir(projPath);
+
+  return { slug };
```

Comment:
I think we should return as much data as possible when we do this. Let's grab the entire return from store.addProject and return it here too.